scene/gui/reference_rect.h

```diff
@@ -41,12 +41,16 @@ class ReferenceRect : public Control {
 protected:
 	void _notification(int p_what);
 	static void _bind_methods();
+	bool show_in_game;
```

Comment:
I think, no need to make it protected, since there is no derived classes for ReferenceRect are exist, and there is almost no performance penalty to call set_show_in_game/get_show_in_game